app/router.js

```diff
@@ -7,6 +7,7 @@ export default class Router extends EmberRouter {
 }
 
 Router.map(function () {
+  // QUESTION: Why the need to reset the namespace?
```

Comment:
Good catch `resetNamespace` is not needed here. Removed on this [PR](https://github.com/rathishvbr/app-one/commit/cf93f6867461c6047e02b14073df9dafa89de760)